llvm/test/CodeGen/Generic/live-debug-label.ll

```diff
@@ -1,5 +1,6 @@
 ; RUN: llc < %s -stop-after=virtregrewriter -o - | FileCheck %s
-;
+; RUN: llc --try-experimental-debuginfo-iterators < %s -stop-after=virtregrewriter -o - | FileCheck %s
```

Comment:
Do this and the other test updates need to be done now that RemoveDIs is on by default? If we do keep this, I think we'd need one with `--experimental-debuginfo-iterators=false` and one with `true` to ensure we're covering both paths.

Reply:
I mention this in the patch summary - my thinking is that this preserves coverage if we need to disable RemoveDIs temporarily. Roping in @jmorse to see if he's got an opinion on testing strategy going forward.

>I think we'd need one with --experimental-debuginfo-iterators=false

However, I don't think this question (above) is for this patch, as that could arguably be applied to all tests with debug info?

Reply:
Yeah, it doesn't necessarily need to be added here - the important part is just that adding `--try-experimental-debuginfo-iterators` is unnecessary/meaningless now, since we're already using it by default.

Reply:
The (offline) consensus agrees with your position -- I've removed them from the patch.